src/service/print/docxHelper.js

```diff
@@ -270,20 +275,20 @@ async function renderMCQ(question, questionCounter, marks) {
       typeof questionOptions[0][0] === "object"
     ) {
       if (questionOptions[0][0].text) {
-        questionOpt.push(["I.", questionOptions[0][0].text[1]]);
+        questionOpt.push([`${getLanguageKey(printLanguage, 'i')+")"}`, questionOptions[0][0].text[1]]);
         imageProperties.push({
           width: 0,
           height: 0,
         });
       } else {
-        questionOpt.push(["I.", questionOptions[0][0].image]);
+        questionOpt.push([`${getLanguageKey(printLanguage, 'i')+")"}`, questionOptions[0][0].image]);
         imageProperties.push({
           width: questionOptions[0][0].width,
           height: questionOptions[0][0].height,
         });
       }
     } else {
-      questionOpt.push(["I.", questionOptions[0][0]]);
+      questionOpt.push([`${getLanguageKey(printLanguage, 'i')+")"}`, questionOptions[0][0]]);
```

Comment:
Why can't you move the entire this logic into different function as pass the index to it
for loop 
   setOption(index, questionOptions)

```
function setOption(index, questionOptions)
if (questionOptions[index] !== undefined) {
    if (
      questionOptions[index][0] !== undefined &&
      typeof questionOptions[index][0] === "object"
    ) {
      if (questionOptions[index][0].text) {
        questionOpt.push([`${getLanguageKey(printLanguage, index)+")"}`, questionOptions[index][0].text[1]]);
        imageProperties.push({
          width: 0,
          height: 0,
        });
      } else {
        questionOpt.push([`${getLanguageKey(printLanguage, index+1)+")"}`, questionOptions[index][0].image]);
        imageProperties.push({
          width: questionOptions[index][0].width,
          height: questionOptions[index][0].height,
        });
      }
    } else {
      questionOpt.push([`${getLanguageKey(printLanguage, index+1)+")"}`, questionOptions[index][0]]);
      imageProperties.push({
        width: 0,
        height: 0,
      });
    }
  }
```

Reply:
This way if there are only 2 option or more than 4 also the code will works.

Reply:
This is done @vinukumar-vs

Reply:
Shall I resolve this conversation, as this is implemented too?